client/python/gradio_client/types.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
Ah nice thanks for bringing this out. @pngwn will be happy. Maybe we even more it two directories up to the `client` folder since relevant to all of the client